model/LongRunningWorker.php

```diff
@@ -110,6 +108,7 @@ public function run()
                     continue;
                 }
 
+                ++$this->iterations;
```

Comment:
The problem with moving this here is that if there are no tasks in the queue this worker will be alive forever even if the `--limit` is not 0.

I think we need to rethink this part

Reply:
I can revert it back - it will work properly now without this change 🙂

Reply:
I think we will need to do it indeed :) Thanks!